ludwig/utils/data_utils.py

```diff
@@ -1089,3 +1089,17 @@ def use_credentials(creds):
                     json.dump(old_conf, f)
                 conf.clear()
                 set_conf_files(tmpdir, conf)
+
+
+def get_sanitized_feature_name(feature_name: str) -> str:
```

Comment:
nit: We should stick with either `feature` or `column` in these function names

Reply:
Based on the discussion at standup today, I would say that current names seem ok. `get_sanitized_feature_name()` is called on a feature-wise basis while `sanitize_column_names()` is used for the underlying dataframe. They share the same sanitization step.

Reply:
Agreed! It was good to get some clarification on the nomenclature